templates/resources/common/modify-wrapper-linux-conf.gawk

```diff
@@ -37,3 +37,13 @@ END {
   printf("wrapper.java.additional.%d=-XX:+UseCGroupMemoryLimitForHeap\n", additionals + 2);
   printf("wrapper.java.additional.%d=-XX:MaxRAMFraction=2\n", additionals + 3);
 }
+
+# Enable JMX exporter for Prometheus
+END {
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote\n", additionals + 4);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.authenticate=false\n", additionals + 5);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.ssl=false\n", additionals + 6);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.port=9011\n", additionals + 7);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.rmi.port=9011\n", additionals + 8);
```

Comment:
Do we need to open up both RMI and regular port?

Reply:
@hierynomus yep. This are default settings. If we don't specify **jmxremote.rmi.port** , then on each startup random port will be choosen.

Reply:
ok!